crates/astria-sequencer-relayer/local.env.example

```diff
@@ -1,3 +1,13 @@
+# A list of filter directives of the form target[span{field=value}]=level.
+ASTRIA_SEQUENCER_RELAYER_LOG=astria_sequencer_relayer=info
```

Comment:
Should this have quotes? `ASTRIA_SEQUENCER_RELAYER_LOG="astria_sequencer_relayer=info"`

Reply:
No, that's not necessary:

```sh
$ export FOO=bar
$ echo $FOO
bar
```